src/index.html

```diff
@@ -20,8 +21,369 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="body">
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="Moyo_logo"
+          class="header__logo"
+        />
+      </a>
 
-  <body>
-    <h1>Catalog</h1>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link nav__link--is-active is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
```

Comment:
All navigation links currently point to 'index.html'. If these links are intended to navigate to different sections or pages, ensure each 'href' attribute is updated accordingly.